lib/measures/tbd/measure.rb

```diff
@@ -59,7 +59,7 @@ def run(model, runner, user_arguments)
       return false
     end
 
-    surfaces = processTBD(model, set)
+    io, surfaces = processTBD(model, set)
     surfaces.each do |id, surface|
```

Comment:
For JSON export, the easiest and safest solution is to prune/edit the _io_ hash once the derated _surfaces_ objects have their _:ratio_ values calculated (and therefore right before both objects are returned at the end of processTBD). Initial tests seem to work just fine.